### ISSUE TITLE
Upgrade from C119 -> C120 on Android ARM x64 via 1.62.x - Nightly (Checking C120)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#34490]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [ ] Rewards
    - [ ] Wallet balance is retained
    - [ ] Auto-contribute list is retained
    - [ ] Both Tips and Monthly Contributions are retained
    - [ ] Wallet panel transactions list is retained
    - [ ] Changes to rewards settings are retained
    - [ ] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [ ] Ads
    - [ ] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [ ] Changes to ads settings are retained
    - [ ] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [ ] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Pixel 6` running `Android 14` using the following build(s):

```
Brave | 1.62.73 Chromium: 120.0.6099.28 (Official Build) canary (32-bit)
--- | ---
Revision | 522e9147d931744b1641084046c197caf7b341f0
OS | Android 14; Build/AP11.231020.013.A1; 34; REL
```

Quick note that the rewards/ads portion can't be checked due to https://github.com/brave/brave-browser/issues/34290 still being an issue/not fixed. Hence, not being able to opt-in into rewards on the Nightly channel.

![Screenshot_20231127-011519](https://github.com/brave/brave-browser/assets/2602313/88d8b799-7225-4db3-8296-bf617fde3d24)
